tests/test_tables.py

```diff
@@ -161,13 +162,35 @@ def test_get_current_date_from_bad_coupler_res(self):
         with self.assertRaises(ConfigError):
             _get_current_date_from_coupler_res(coupler_res_filename)
 
-    def test_get_current_date_from_config(self):
-        config = DEFAULT_CONFIG.copy()
+    def test_get_current_date_from_config_force_date_true(self):
+        config = copy.deepcopy(DEFAULT_CONFIG)
         config["namelist"]["coupler_nml"]["force_date_from_namelist"] = True
         config["namelist"]["coupler_nml"]["current_date"] = valid_current_date
         current_date = get_current_date(config)
         self.assertEqual(current_date, valid_current_date)
 
+    def test_get_current_date_from_config_force_date_false(self):
```

Comment:
If it's not too much trouble, could you add a case where `initial_conditions` is an empty list `[]`? This is useful for idealized cases which need no initial conditions data, and is an interesting edge case.

Reply:
Case added.